models/topics.model.js

```diff
@@ -49,3 +50,30 @@ exports.fetchComments = async (article_id) => {
     return allPromises[1].rows;
   }
 };
+
+exports.addComment = async (article_id, username, body) => {
+  const queryStr = `INSERT INTO comments (article_id, author, body, created_at) VALUES($1, $2, $3, NOW()) RETURNING comment_id, article_id, author, body, created_at`;
```

Comment:
The date should be automatically created by the column data type when the insertion happens